packages/bpk-mixins/src/mixins/_layers.scss

```diff
@@ -74,19 +74,24 @@
 ///     @include bpk-layer-arrow-top(1rem);
 ///   }
 
-@mixin bpk-layer-arrow-top($size) {
+@mixin bpk-layer-arrow-top($arrow-size) {
   bottom: 100%;
-  left: 50%;
+  width: ($arrow-size * 2);
+  height: $arrow-size;
 
   &::before {
-    top: -($size + $bpk-one-pixel-rem);
-    margin-left: -($size + $bpk-one-pixel-rem);
```

Comment:
Previous we were positioning the arrow at 50% from left - since Popper takes care of this now we dont need to offset.